lib/observer/doc/src/observer_ug.xml

```diff
@@ -340,19 +340,154 @@
+            ,node
+            }).
+
+         start() ->
+            start([]).
+
+         start(Args) ->
+            wx_object:start(?MODULE, Args, []).
+
+         start_link() ->
+            start_link([]).
+
+         start_link(Args) ->
+            wx_object:start_link(?MODULE, Args, []).
+
+         init(Node) ->
+            TNode = case Node of
+                        X when is_atom(X) -> X;
+                        _  -> erlang:node() % default to current node
+                     end,
+            erlang:display(TNode),
+            Parent = wx:new(),
+            process_flag(trap_exit, true),
+
+            Text = atom_to_list(?MODULE),
+            MiniFrame = wxMiniFrame:new(Parent, ?wxID_ANY, Text, [{style,
+                                    ?wxDEFAULT_FRAME_STYLE bor
+                                    ?wxFRAME_FLOAT_ON_PARENT}]),
+            Panel = wxPanel:new(MiniFrame, []),
+
+            % Add a button to  restart all application of given node
+            Sz = wxBoxSizer:new(?wxVERTICAL),
+            ButtSz = wxStaticBoxSizer:new(?wxHORIZONTAL, Panel,
+                     [{label, atom_to_list(TNode)}]),
+            SzFlags = [{proportion, 0}, {border, 4}, {flag, ?wxALL}],
+            B  = wxButton:new(Panel, 10, [{label,"Restart applications"}]),
+            wxSizer:add(ButtSz, B, SzFlags),
+
+            wxMiniFrame:setSize(MiniFrame, {150,100}),
+            wxMiniFrame:center(MiniFrame),
+            wxMiniFrame:show(MiniFrame),
+            wxWindow:connect(Panel, command_button_clicked),
+            wxWindow:setSizer(Panel, Sz),
+            wxSizer:layout(Sz),
+            wxWindow:refresh(Panel),
+            {MiniFrame, #state{frame=MiniFrame ,node = TNode}}.
+
+         %%%%%%%%%%%%
+         %% Callbacks
+
+         handle_info({'EXIT',_, wx_deleted}, State) ->
+            {noreply,State};
+         handle_info({'EXIT',_, shutdown}, State) ->
+            {noreply,State};
+         handle_info({'EXIT',_, normal}, State) ->
+            {noreply,State};
+         handle_info(_, State) ->
+            {noreply,State}.
+
+         handle_call(_, _From, State) ->
+            {reply,ok,State}.
+
+         handle_cast(_, State) ->
+            {noreply,State}.
+
+         handle_event({wx,10, _, _,
+                     {wxCommand,command_button_clicked,_,_,_}}, State) ->
+            % Restart applications on node
+            Res = rpc:call(State#state.node, init, restart, []),
+            io:format("restarting application on ~p : ~p~n", [State#state.node, Res]),
+            {noreply,State};
+         handle_event(Ev,State) ->
+            io:format("~p Got uncaught event ~p ~n",[?MODULE, Ev]),
+            {noreply, State}.
+
+         code_change(_, _, State) ->
+            {stop, not_yet_implemented, State}.
+
+         terminate(_Reason, _State = #state{frame=Frame}) ->
+            wxMiniFrame:destroy(Frame),
+            %wx:destroy() % DO NOT or observer will be closed too
```

Comment:
Like:
```xml
<warning>
  <p>To prevent the observer from being closed, please do not call
  <c>wx:destroy()</c> from your plugins.</p>
</warning>
```